roles/bats/tasks/main.yml

```
@@ -43,6 +43,7 @@
       register: "test_output"
       ignore_errors: true
       environment: "{{ bats_environment }}"
+      become_flags: '-H -S -n -i'
```

HSn is Ansible's default, we're adding `-i` to it.

---

I always like long options
```suggestion
      become_flags: '--set-home --stdin --non-interactive --login'
```

---

I thought about that, but then erred on the side of a smaller diff (ansible uses short opts internally).

We could do `-H -S -n --login`?